docs/quickstart.md

```diff
@@ -1,150 +1,160 @@
 
-### Ingest Metadata
-
-To start pushing your company's metadata into DataHub, take a look at [UI-based Ingestion Guide](./ui-ingestion.md), or to run ingestion using the cli, look at the [Metadata Ingestion Guide](../metadata-ingestion/README.md).
+```json
+username: datahub
+password: datahub
+```
 
-### Invite Users
+To change the default credentials, please refer to [Change the default user datahub in quickstart](authentication/changing-default-credentials.md#quickstart).
 
-To add users to your deployment to share with your team check out our [Adding Users to DataHub](authentication/guides/add-users.md)
+### Ingesting Sample Data
 
-### Enable Authentication
+To ingest the sample metadata, run the following CLI command from your terminal
 
-To enable SSO, check out [Configuring OIDC Authentication](authentication/guides/sso/configure-oidc-react.md) or [Configuring JaaS Authentication](authentication/guides/jaas.md).
+```bash
+datahub docker ingest-sample-data
+```
 
-To enable backend Authentication, check out [authentication in DataHub's backend](authentication/introducing-metadata-service-authentication.md#configuring-metadata-service-authentication).
+:::note
 
-### Change the Default `datahub` User Credentials
+If you've enabled [Metadata Service Authentication](authentication/introducing-metadata-service-authentication.md), you'll need to provide a Personal Access Token
+using the `--token <token>` parameter in the command.
 
-:::note
-Please note that deleting the `Data Hub` user in the UI **WILL NOT** disable the default user. You will still be able to log in using the default 'datahub:datahub' credentials. To safely delete the default credentials, please follow the guide provided below.
 :::
 
-Please refer to [Change the default user datahub in quickstart](authentication/changing-default-credentials.md#quickstart).
+That's it! Now feel free to play around with DataHub!
 
-### Move to Production
+## Next Steps
+
+- [Quickstart Debugging Guide](./troubleshooting/quickstart.md)
+- [UI-based Ingestion Guide](./ui-ingestion.md)
+- [Metadata Ingestion Guide](../metadata-ingestion/README.md)
+- [Adding Users to DataHub](authentication/guides/add-users.md)
+- [Configuring OIDC Authentication](authentication/guides/sso/configure-oidc-react.md)
+- [Configuring JaaS Authentication](authentication/guides/jaas.md)
+- [authentication in DataHub's backend](authentication/introducing-metadata-service-authentication.md#configuring-metadata-service-authentication).
+- [Change the default user datahub in quickstart](authentication/changing-default-credentials.md#quickstart)
 
 We recommend deploying DataHub to production using Kubernetes. We provide helpful [Helm Charts](https://artifacthub.io/packages/helm/datahub/datahub) to help you quickly get up and running. Check out [Deploying DataHub to Kubernetes](./deploy/kubernetes.md) for a step-by-step walkthrough.
 
+<details>
+<summary>Reasons to move to production</summary>
```

Comment:
do you think this deserves it's own `### Move to Production` section?

also, we should use a docusaurus caution admonition when saying "Don't use quickstart in production settings"

Reply:
Yup moved this to the last part & added caution